CHANGELOG/CHANGELOG-1.27.md

```diff
@@ -198,6 +200,15 @@ name | architectures
 
 ## Changelog since v1.26.0
 
+## Known Issues
+
+### The PreEnqueue extension point doesn't work for Pods going to activeQ through backoffQ
+
+In v1.26.0, we've found the bug that the PreEnqueue extension point doesn't work for Pods going to activeQ through backoffQ. 
+It doesn't affect any of the vanilla Kubernetes behavior, but, may break custom PreEnqueue plugins.
+
+The cause PR is [reverted](https://github.com/kubernetes/kubernetes/pull/117194) by v1.26.1.
```

Comment:
I think this should mention v1.27.0 and v1.27.1 if I'm not mistaken.

Reply:
fixed